packages/pds/src/api/com/atproto/server/getSession.ts

```diff
@@ -37,16 +50,38 @@ export default function (server: Server, ctx: AppContext) {
 
       return {
         encoding: 'application/json',
-        body: {
+        body: render(auth, {
```

Comment:
At a glance the term "render" made me wonder if some other representation was being constructed, e.g. html.  Not sure if others would have the same association, though.  I also like terms "presentation" (a term used in the appview) or "output" (from lexicon parlance).